plugins/linear-genome-view/src/LinearGenomeView/components/OverviewHighlight.tsx

```diff
@@ -63,24 +64,28 @@ const OverviewHighlight = observer(function OverviewHighlight({
     return null
   }
 
-  const asm = assemblyManager.get(model.highlight?.assemblyName)
+  return model.highlight
```

Comment:
multiple checks for model.highlight being "falsey". the lines above it check this. also, if we know it is an array, falsey check may be unneeded